src/utils/gameDataConversions.js

```diff
@@ -19,8 +23,10 @@ const gameDataConversions = (array) => {
 		if (isNaN(game.maxplaytime[0].$.value))
 			game.maxplaytime[0].$.value = '--';
 
-		if (game.yearpublished[0].$.value === undefined)
-			game.yearpublished[0].$.value = ['--'];
+		if (collectionData[game.$.id].yearpublished)
+			game.yearpublished[0].$.value = collectionData[game.$.id].yearpublished[0];
+		if (game.yearpublished[0].$.value === undefined || game.yearpublished[0].$.value == 0)
```

Comment:
The compiler warns `Line 28:84:  Expected '===' and instead saw '=='`, can you please add another `=` to make it happy? :)

Reply:
fixed